core/editor/config-providers/editor-common-configs.php

```diff
@@ -114,6 +114,14 @@ public static function get_script_configs() {
 					'domain' => 'elementor',
 				],
 			],
+			[
+				'handle' => 'elementor-responsive-bar',
+				'src' => '{{ELEMENTOR_ASSETS_URL}}js/responsive-bar{{MIN_SUFFIX}}.js',
+				'deps' => [ 'elementor-editor' ],
+				'i18n' => [
+					'domain' => 'elementor',
+				],
```

Comment:
Can you validate that the translations works?

Reply:
Works :D